solr/CHANGES.txt

```diff
@@ -110,6 +110,9 @@ Improvements
 
 * SOLR-14114: Add WARN to Solr log that embedded ZK is not supported in production (janhoy)
 
+* SOLR-14281: Make sharedLib configurable through SysProp and allow multiple comma separated paths.
```

Comment:
SysProp should by "sys-prop" or system property.  The way you capitalized it makes it seem like some class.

Reply:
Re-phrased